pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/query/UploadStatus.kt

```diff
@@ -67,7 +67,7 @@ class UploadStatus {
             // Convert the reports to their schema objects
             val reportsWithSchemaPairs = mutableListOf<Pair<SchemaDefinition, ReportDao>>()
             reports.forEach { report ->
-                if (report.contentType != "json")
+                if (report.contentType != "application/json")
```

Comment:
Make is so contentType can be "json" or "application/json".

Reply:
Updated according to the suggestions